patches/minecraft/net/minecraft/client/renderer/block/model/ItemTransforms.java.patch

```diff
@@ -54,7 +52,12 @@
 +         for (TransformType type : TransformType.values()) {
 +            if (type.isModded()) {
 +               var transform = this.m_111815_(p_111822_, jsonobject, type.getSerializeName());
-+               if (transform != ItemTransform.f_111754_) {
```

Comment:
Is there any concern about order here for fallbacks? Might need to document that a fallback must be an earlier element in the enum, or require fallbacks to be vanilla.

Reply:
Either the key exists in the json, or it doesn't. The ordering is irrelevant... unless I missed something.

Reply:
I could also have missed something. Do you process fallbacks at the same time as you serialize, or do fallbacks in a second pass? In other words, is it possible when I see the key does not exist (and need a fallback) that the fallback has not yet parsed?

Reply:
The code doesn't use existing parsed data, it just re-parses the same data for the new key. The fallback is only used to decide which key to read from.